CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking changes
 - Bareos 25 disables SSL on the PostgreSQL connection, as we have observed strange issues with SSL enabled.[Issue #1965]
-- Bareos 25 and later will not modify or extend existing configuration directories on package updates [PR #2338]
+- Bareos 25 and later will not modify or extend existing configuration directories on package updates [PR #2338]<br>
   Note: on FreeBSD unmodified configuration changes from Bareos <= 24 will get removed and
   replaced by the new default configuration of the current package.
```

Comment:
Instead having html code here, we might be interested by using the quote marker
```suggestion
- Bareos 25 and later will not modify or extend existing configuration directories on package updates [PR #2338]
> Note: on FreeBSD unmodified configuration changes from Bareos <= 24 will get removed and
replaced by the new default configuration of the current package.
```
I've tested by entering another list entry afterward and the rendering is ok.